int16 right_forward
int16 right_side
int16 left_side
int16 lefot_forward
int16 sum_all
int16 sum_forward

